docs/pages/access-controls/reference.mdx

```diff
@@ -357,9 +357,41 @@ allow:
     - resources:
         - token
       verbs: [list, create, read, update, delete]
+```
+
+### Allowing access to token resources
 
+If you configure a role that allows tokens to be created, users assigned to the 
```

Comment:
I personally prefer "users with the role" over "users assigned to the role" but will leave this one up to you.

From a UI perspective, you select a user, and then assign roles to that user. (Roles are assigned to users, not the other way around)